softlayer/resource_softlayer_scale_group.go

```diff
@@ -245,9 +245,8 @@ func buildScaleVlansFromResourceData(d *schema.Set, meta interface{}) ([]datatyp
 
 		if len(networkVlans) < 1 {
 			return nil, fmt.Errorf(
-				"Unable to locate a vlan matching the provided router hostname and vlan number: %s/%s",
-				primaryRouterHostname,
-				vlanNumber)
+				"Unable to locate the specified vlan: %d.%s",
+				vlanNumber, primaryRouterHostname)
```

Comment:
Yes, but in 1) removing the field names and 2) choosing a delimiter that appears in your data 100% of the time, it's now ambiguous.  It may be more recognizable to us, but there is no such thing as vlan '1234.bcr01a.sjc03'.  It isn't a thing, and a user new to SoftLayer will find this confusing.